network_analysis/service-area-task/src/main/resources/service_area_task.fxml

```diff
@@ -15,31 +15,55 @@
   ~ the License.
   -->
 
+<?import javafx.geometry.Insets?>
 <?import javafx.scene.control.Button?>
 <?import javafx.scene.control.ProgressIndicator?>
 <?import javafx.scene.control.ToggleButton?>
 <?import javafx.scene.control.ToggleGroup?>
-<?import javafx.scene.layout.StackPane?>
 <?import javafx.scene.layout.HBox?>
+<?import javafx.scene.layout.StackPane?>
+<?import javafx.scene.layout.VBox?>
 <?import com.esri.arcgisruntime.mapping.view.MapView?>
-
-<StackPane fx:controller="com.esri.samples.service_area_task.ServiceAreaTaskController" xmlns:fx="http://javafx.com/fxml"
+<?import javafx.scene.layout.Background?>
+<?import javafx.scene.layout.BackgroundFill?>
+<?import javafx.scene.paint.Color?>
+<StackPane fx:controller="com.esri.samples.service_area_task.ServiceAreaTaskController"
+           xmlns:fx="http://javafx.com/fxml"
            stylesheets="/style.css">
-           
+
     <!--SDK SceneView-->
     <MapView fx:id="mapView"/>
     <!-- For loading service areas -->
     <ProgressIndicator fx:id="progressIndicator" maxHeight="50.0" maxWidth="50.0" visible="false"/>
-    
-    <!-- Top bar button select -->
-    <HBox StackPane.alignment="TOP_LEFT" maxWidth="200" maxHeight="50">
-      <ToggleButton fx:id="btnAddFacility" text="Add Facilities" minWidth="200">
-      <toggleGroup>
-        <ToggleGroup fx:id="toggleGroup"/>
-        </toggleGroup>
-      </ToggleButton>
-      <ToggleButton fx:id="btnAddBarrier" toggleGroup="$toggleGroup" text="Add Barrier"  onAction="#createBarrier" minWidth="200"/>
-      <Button text="Show Service Areas" onAction="#showServiceAreas" minWidth="200"/>
-      <Button text="Reset" onAction="#clearRouteAndGraphics" minWidth="200"/>
-    </HBox>  
-</StackPane>
\ No newline at end of file
+
+    <!-- Controls area -->
+    <VBox StackPane.alignment="TOP_LEFT" maxWidth="180" maxHeight="50" spacing="10">
+        <padding>
+            <Insets topRightBottomLeft="10"/>
+        </padding>
+
+        <background>
+            <Background>
+                <fills>
+                    <BackgroundFill>
+                        <fill>
+                            <Color opacity="0.3"/>
+                        </fill>
+                    </BackgroundFill>
+                </fills>
+            </Background>
+        </background>
+
+        <HBox>
+            <ToggleButton fx:id="btnAddFacility" text="Add Facilities" minWidth="90">
+                <toggleGroup>
+                    <ToggleGroup fx:id="toggleGroup"/>
+                </toggleGroup>
+            </ToggleButton>
+            <ToggleButton fx:id="btnAddBarrier" toggleGroup="$toggleGroup" text="Add Barrier" onAction="#createBarrier"
+                          minWidth="90"/>
+        </HBox>
+        <Button text="Show Service Areas" onAction="#showServiceAreas" minWidth="180"/>
+        <Button text="Reset" onAction="#clearRouteAndGraphics" minWidth="180"/>
```

Comment:
Is there a way we can clean up all of these min/max widths? Not a big deal if not.

Reply:
Good point, I've gone ahead and put all the styling in the CSS file. Perhaps we can do this with samples going forward? Makes the FXML look nice and clean.